cmd/run.go

```diff
@@ -164,7 +168,7 @@ func runCommand(cmd *cobra.Command, args []string) {
 				limitFlag = len(info)
 			}
 
-			if err = ssmx.RunInvocations(sess, svc, info[:limitFlag], params, dryRunFlag, completedInvocations); err != nil {
+			if err = ssmx.RunInvocations(sess, svc, info[:limitFlag], params, dryRunFlag, maxConcurrencyFlag, maxErrorsFlag, completedInvocations); err != nil {
```

Comment:
Thankfully, in my PR we've moved building the SendCommandInput to the main function, so we don't have to pass a hundred args. :p

Reply:
I'll rebase when yours gets merged